pallets/automation-time/src/tests.rs

```diff
@@ -38,12 +38,16 @@ const LAST_BLOCK_TIME: u64 = START_BLOCK_TIME / 1_000;
 #[test]
 fn schedule_invalid_time() {
```

Comment:
Since all timeAutomation call has two kinds of scheduling, Fixed and Recurring, we should duplicate **time related tests** and **test both types**.

For Fixed, multiple timestamps need always to be tested since the input is an array, Vec<UnixTime>
For Recurring, only one timestamp needs to be tested, but there’s a second parameter `frequency` to test as well, `{ next_execution_time: UnixTime, frequency: Seconds}`.